exonum/src/helpers/fabric/maintenance.rs

```diff
@@ -27,8 +27,11 @@ use node::NodeConfig;
 use super::internal::{CollectedCommand, Command, Feedback};
 use super::{Argument, CommandName, Context};
 
+// Conext entry for the path to the node config.
```

Comment:
Typo

Reply:
Indeed. Interesting that `cspell` didn't catch it in Travis run.